app/client/lib/highlight.js

```diff
@@ -1,5 +1,4 @@
 var find = require('findandreplacedomtext')
-var css = require('dom-css')
 
 // Highlights all the words in text inside a node
 function Highlight (node, text, wrapper) {
```

Comment:
`text` can now be `null`, in the case where the search has tags but nothing else.

To handle this we just need to:

```js
if (!text) return
```

And just in case, we should also do something if `node` doesn't exist:

```js
if (!node) throw new Error('highlight called without a node', arguments)
```